python/src/puml/helpers/requirements_helpers.py

```diff
@@ -0,0 +1,132 @@
+from typing import List, Optional
+
+
+def _get_requirements_defs(reqs: dict) -> List[dict]:
+    """
+    Helper method for extracting and sorting data from a requirement specification
+
+    Args:
+        reqs (dict): A dictionary of requirement definitions
+
+    Returns:
+        A List of dictionaries of organized data for diagram generation
+    """
+    requirements = []
+    if reqs:
+        for req_key in reqs:
+            req = reqs[req_key]
+            id = req.structure["req"]["id"]
+            attributes = req.structure["req"]["attributes"]
+            type = _get_requirement_type(attributes)
+            title = None
+            name = req.structure["req"]["name"]
```

Comment:
Based on the core definitions in `aac.aac`, the title and name are the same.
```
schema:
  name: Requirement
  package: aac.lang
  root: req
  fields:
    - name: name
      type: string
      description: |
        The name or title of the requirement.
      is_required: true
```

```suggestion
            title = req.structure["req"]["name"]
            name = req.structure["req"]["name"]
```

Reply:
It was left as None in the original code for some reason, but I made the suggested change.

Reply:
Because this code originally came from attempts at working with legacy 0.3.x, and we didn't have that inclusion of them being the same in the previous core structure is all.